### PR TITLE
EPMRPP-81532 || Fix displaying colon to separate attributes

### PR DESCRIPTION
**Pull request by AmsterGet**

### CODE REVIEW

File: app/src/common/utils/attributeUtils.js [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 export const getAttributeValue = ({ key, value } = {}) => value || key;
 
-export const formatAttribute = ({ key, value } = {}, divider = ':') => {
+export const attributeFormatter = ({ key, value } = {}, divider = ':') => {
```
Comment (Bam6ycha) [inline app/src/common/utils/attributeUtils.js:19]: As for me default value for the divider looks strange here.
Do we really need it?

File: app/src/common/utils/attributeUtils.js [hunk 2]
```diff
@@ -26,8 +26,12 @@ export const formatAttribute = ({ key, value } = {}, divider = ':') => {
   return getAttributeValue({ key, value }) || '';
 };
 
+export const formatAttribute = (attribute) => {
```
Comment (Bam6ycha) [inline app/src/common/utils/attributeUtils.js:29]: And the name for this function does not look consistent.
We have `formatAttributeWithSpacedDivider`. May be better to rename it to `formatAttributeWithDefaultDivider`?

**Review by Bam6ycha — CHANGES_REQUESTED** [review]

File: app/src/common/utils/attributeUtils.js (hunk 1 above)
Comment (AmsterGet) [reply app/src/common/utils/attributeUtils.js:19]: Technically this function can be called without `divider`.
But I suppose we should not use it as is, only through `formatAttribute...` functions.
I'll remove it.

File: app/src/common/utils/attributeUtils.js (hunk 2 above)
Comment (AmsterGet) [reply app/src/common/utils/attributeUtils.js:29]: Usually when we use smth without specifiers it means that we expect the default behavior.
So here we have `formatAttribute` - the default way to format attributes (and the frequently used) and the `formatAttributeWithSpacedDivider` which provides non-default behavior for some specific use cases.
Another reason why I decided to leave the default naming as is - to not change the API for plugins where this utility used as well.

Comment (Bam6ycha) [reply app/src/common/utils/attributeUtils.js:29]: I got it!

**Review by Bam6ycha — APPROVED** [review]

**Pull request merged**